cluster-scope/overlays/nerc-ocp-prod/feature/external-console-access/routes/api.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: route.openshift.io/v1
+kind: Route
+metadata:
+  name: api-external
```

Comment:
I am confused, there's a route (cluster-scope/overlays/nerc-ocp-prod/feature/external-console-access/api-proxy/route.yaml) with the same name but to the service `api-proxy`. This one is to `apiserver` Service. Both have the same host though `api.apps.shift.nerc.mghpcc.org`.

Additionally, this is not included in the kustomization file in this directory.